ROADMAP.md

```diff
@@ -1,45 +1,42 @@
-# Roadmap
+# Apollo iOS Roadmap - _Last Updated July 2021_
 
-This document is meant to give the community some idea of where we're going with the iOS SDK in the short and longer term. 
+_If this document has not been updated within the past three months, please [file an issue](https://github.com/apollographql/apollo-ios/issues/new/choose) asking the [maintainers](https://github.com/apollographql/apollo-ios#maintainers) to update it._
 
-This document was last updated on January 4th, 2020 
+Releases adhere to the [Semantic Versioning Specification](https://semver.org/). Under this scheme, version numbers and the way they change convey meaning about the underlying code and what has been modified from one version to the next.
 
-^ If that's more than three months ago, please file an issue asking [@designatednerd](https://github.com/designatednerd) to update this document. 😃
+## 0.x - _Current_
 
+This version is being used in many Production codebases, and we're committed to resolving issues and bugs raised by the community. We are not considering any further substantial work to be done in this version.
 
-## Short-Term
+## 1.0 - _Estimated Release Q4 2021_
 
-These are things we plan to be working on **within the next 3 months**.
+These are the three guiding principles we're aiming for in a 1.0 release.
+- **Stability**: Achieve a stable foundation that can be trusted for years to come, maintaining backwards compatibility within major version releases.
+- **Completeness**: The three main parts to the SDK must be feature complete: code generation, network fetching/parsing, and caching.
```

Comment:
@AnthonyMDev, @designatednerd - I'm not a fan of saying "feature complete" here. IMO I think it'd be better to state "MVP complete" or "first-generation complete". Thoughts?

Reply:
Caching is certainly not going to be feature complete with 1.0. I don't think any of it is going to be "feature complete" really. I have ideas for 100s of features I want to add after 1.0, and many of them are, in my opinion, pretty vital pieces. I'd personally just remove this whole "Completeness" line.

Reply:
> Caching is certainly not going to be feature complete with 1.0. I don't think any of it is going to be "feature complete" really. I have ideas for 100s of features I want to add after 1.0, and many of them are, in my opinion, pretty vital pieces.

That's why I wanted to be explicit about "not complete but at a state where we believe it's good enough to satisfy production 1.0 uses".

> I'd personally just remove this whole "Completeness" line.

I think there are expectations re. cache updates for 1.0 and we should be explicit about why we're not focused on cache features for 1.0. Stating "MVP" or "first-generation" gives us an intentional way to talk about that.

Reply:
Maybe something more like "A good foundation for the addition of future features without breaking what exists"